#!/usr/bin/env python2

a = ['a', 'b', 'c']
b = [1, 2, 3]
c = range(5)
#sum(a) # causes exception
print sum(b)
print sum(c)
